src/database/seeds/Participants.ts

```diff
@@ -101,8 +101,22 @@ export async function CreateParticipant(
 }
 
 export async function seed(knex: Knex): Promise<void> {
-  // Deletes ALL existing entries
-  await knex('participants').whereILike('name', '%example').del();
+  // Deletes existing entries for sampleData
+
+  await knex('users')
```

Comment:
Maybe extract `sampleData.map((d) => d.name)` to a variable `sampleParticipantNames`?